packages/rs-drive-abci/src/platform_types/epoch_info/v0/mod.rs

```diff
@@ -133,13 +130,14 @@ impl EpochInfoV0 {
         genesis_time_ms: u64,
         block_time_ms: u64,
         previous_block_time_ms: Option<u64>,
+        epoch_time_s: u64,
     ) -> Result<EpochInfoV0, Error> {
         let previous_block_time = match previous_block_time_ms {
             Some(block_time) => block_time,
             None => return Ok(EpochInfoV0::default()),
         };
 
-        let epoch_change_time = Decimal::from(EPOCH_CHANGE_TIME_MS_V0);
+        let epoch_change_time = Decimal::from(epoch_time_s * 1000);
```

Comment:
Let's use a constant

Reply:
why? s to ms is 1000, no need for a constant.

Reply:
(as it will never change).

Reply:
I prefer to use constants for any number especially if it’s repeating https://en.wikipedia.org/wiki/Magic_number_(programming)

It's up to you